tools/ramdump/dumpParser.py

```diff
@@ -702,6 +702,109 @@ def main():
 		print '&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&'
 
 
+		print ''
+		print ''
+		print 'Details of Heap Usages (Size in Bytes)'
+		print ''
+
+		g_mmheap = rParser.get_address_of_symbol("g_mmheap")
+
+		# Read config informaiton
+		fd = open("../../os/.config", 'r')
```

Comment:
How about defining PATH for many cases?
If folder structure changed, we can modify the path easily.